src/components/common/DateCell.jsx

```diff
@@ -5,7 +5,14 @@ import { useEffect, useState } from 'react';
 const DAYS_BEFORE_EXPIRATION_WARNING = 90;
 
 export const DateCell = ({ value }) => {
-  const date = DateTime.fromISO(value);
+  console.log(value);
```

Comment:
Console logs should probably come out.
One on line 14 as well.

Reply:
thanks for catching them